packages/paste-core/components/date-picker/__tests__/index.spec.tsx

```diff
@@ -107,6 +107,8 @@ describe('DatePicker', () => {
       const {container} = render(
         <CustomizationProvider
           baseTheme="default"
+          // @ts-expect-error global test variable
```

Comment:
💡 We could prevent this error from being reported by adding an interface for `TestTheme` on our globals object.